### PR TITLE
InstructorResultsPage: "Collapse" button isn't working for participants without responses #4073

### LINKED ISSUE #4073: InstructorResultsPage: "Collapse" button isn't working for participants without responses
As per title

### PR DESCRIPTION
**Pull request by kanghj**
Fixes #4073

### CODE REVIEW

File: src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java [hunk 1]
```diff
@@ -376,4 +383,23 @@ public WebElement getModerateResponseButtonInQuestionView(int qnNo, int response
                              .findElement(By.xpath("table/tbody/tr[" + responseNo + "]/td[6]/form"));
     }
 
+    public void clickInstructorCollapseButton() {
```
Comment (wkurniawan07) [inline src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java]: The method name and the target web element name is not in sync. One of them needs fix.

**Comment by kanghj** [comment]
Changes made

**Comment by kanghj** [comment]
@wkurniawan07

**Comment by wkurniawan07** [comment]
Can merge.

**Pull request merged**